app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,41 @@
  *= require font-awesome
  */
 
+#map-container { 
+  margin-top: 25px;
+  border-color: #fff;
+  border-style: solid;
+  border-radius: 16px 16px;
+  margin-top: 25px;
+  border-width: 7px;
+  box-shadow: 2px 2px 10px #B1B1B1;
+  width: 50px;
+  height: 400px;
+}
+
+#map { 
+  width: 100%;
+  height: 384px;
+
+}
+
+.search-location {
+ background-color: #fff;
+ background-image: none;
+ border: 1px solid #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 1 spaces